resources/lib/menu_functions.py

```diff
@@ -423,6 +423,25 @@ def getCollections(detailsString):
     collections.append(item_data)
 
     item_data = {}
+    item_data['title'] = i18n('tvshows_latest')
+    item_data['media_type'] = 'Episodes'
+    item_data['path'] = ('{server}/emby/Users/{userid}/Items/Latest' +
+                         '?Limit={ItemLimit}' +
+                         '&Recursive=true' +
+                         '&GroupItems=true' +
+                         '&SortBy=DateCreated' +
+                         '&Fields=' + detailsString +
+                         '&SortOrder=Descending' +
+                         '&Filters=IsUnplayed' +
+                         '&IsVirtualUnaired=false' +
+                         '&IsMissing=False' +
+                         '&IncludeItemTypes=Episode' +
+                         '&ImageTypeLimit=1' +
+                         '&format=json')
+    item_data['name_format'] = 'latest_name_format'
+    collections.append(item_data)
+
+    item_data = {}
```

Comment:
can we add an entry like this to the User library entries above, there is an IF up there for tvshows you can add this menu item there also. I am not sure this will work with parent filtering though.